content/influxdb/v2.1/write-data/oss-to-cloud.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Write data from InfluxDB OSS to InfluxDB Cloud
+description: >
+  Use `to()` or `experimental.to()` to write data from InfluxDB OSS to InfluxDB Cloud.
+  Selectively write data or process data before writing it to InfluxDB Cloud.
+menu:
+  influxdb_2_1:
+    name: Write from OSS to Cloud
+    parent: Write data
+weight: 105
+influxdb/v2.1/tags: [write, ]
+---
+
+To write data from InfluxDB OSS to InfluxDB Cloud, use the Flux
+[`to()`](/flux/v0.x/stdlib/influxdata/influxdb/to/) or
+[`experimental.to()`](/flux/v0.x/stdlib/experimental/to/) functions.
+Write data once with a single query execution or use [InfluxDB tasks](/influxdb/v2.1/process-data/)
+to [routinely write data to InfluxDB Cloud](#automate-writing-data-from-influxdb-oss-to-influxdb-cloud).
+
+{{% cloud %}}
+#### InfluxDB Cloud rate limits
+Write requests to InfluxDB Cloud are subject to the rate limits associated with your
+[InfluxDB Cloud pricing plan](/influxdb/cloud/account-management/pricing-plans/).
+{{% /cloud %}}
+
+1.  Query data from InfluxDB OSS.
+2.  _(Optional)_ Filter or process data to write to InfluxDB Cloud.
```

Comment:
maybe link to filter function would be helpful...and/or link process data procedure?